app/services/pages/loader.rb

```diff
@@ -0,0 +1,12 @@
+module Pages
+  class Loader
+    def self.load
+      Pages::Manager.add_page({ :controller => :smart_proxies, :action => :show }, "smart_proxies/show") do |page|
```

Comment:
why does the smart proxy is hard coded here? is it assumed that each page will be hard coded here or rather via an api?